pkgs/tools/misc/turbo/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, git
+, nodejs
+, protobuf
+, protoc-gen-go
+, protoc-gen-go-grpc
+}:
+
+buildGoModule rec {
+  pname = "turbo";
+  version = "1.6.3";
+
+  src = fetchFromGitHub {
+    owner = "vercel";
+    repo = "turbo";
+    rev = "v${version}";
+    sha256 = "csapIeVB0FrLnmtUmLrRe8y54xmK50X30CV476DXEZI=";
+  };
+
+  modRoot = "cli";
+
+  vendorSha256 = "Kx/CLFv23h2TmGe8Jwu+S3QcONfqeHk2fCW1na75c0s=";
+
+  nativeBuildInputs = [
```

Comment:
buildInputs

Reply:
@milahu i've updated this, just curious what is the diference between them? Reading this just made me more confused https://nixos.org/manual/nixpkgs/stable/#variables-specifying-dependencies

Reply:
nativeBuildInputs is needed for cross-compiling

https://discourse.nixos.org/t/use-buildinputs-or-nativebuildinputs-for-nix-shell/8464

https://github.com/NixOS/nixpkgs/issues/19370

here, cross-compiling is done by buildGoModule
and the other inputs dont care about build vs target platform